addons/sale_crm/wizard/crm_opportunity_to_quotation.py

```diff
@@ -35,7 +35,6 @@ def default_get(self, fields):
     action = fields.Selection([
         ('create', 'Create a new customer'),
         ('exist', 'Link to an existing customer'),
-        ('nothing', 'Do not link to a customer')
```

Comment:
What happens if I choose "nothing" without having a customer set ? Does it crash ?

Reply:
Nope, it simply means that User is redirected to quotation with blank partner, and manually have to select / create a partner from there instead of the lead itself.